libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -1067,11 +1067,11 @@ float AC_AttitudeControl::get_althold_lean_angle_max_cd() const
 // Return roll rate step size in centidegrees/s that results in maximum output after 4 time steps
 float AC_AttitudeControl::max_rate_step_bf_roll()
 {
-    float alpha = MIN(get_rate_roll_pid().get_filt_E_alpha(), get_rate_roll_pid().get_filt_D_alpha());
+    float alpha = MIN(get_rate_roll_pid().get_filt_E_alpha(AP::scheduler().get_filtered_loop_time()), get_rate_roll_pid().get_filt_D_alpha(AP::scheduler().get_filtered_loop_time()));
```

Comment:
Here we change to using the filtered loop time?

Reply:
Yes, this function attempts to predict what the next loop will be so we want the average loop time or the target loop time. I am not sure if I chose the correct one yet, I need to think about this a little more.